master/buildbot/newsfragments/fix-buildPushesWithNoCommit-for-merge.bugfix

```diff
@@ -0,0 +1 @@
+:py:class:`~buildbot.changes.GitPoller`: buildPushesWithNoCommits now rebuilds for a known branch that was updated to an existing commit.
```

Comment:
please double backquote (`) ``buildPushesWithNoCommits`` so that the spellchecker is happy